index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Greta Moseley | Portfolio</title>
+		<link href="https://fonts.googleapis.com/css?family=Julius+Sans+One|Marcellus|Marcellus+SC|Nixie+One|Open+Sans" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+	</head>
+	<body>
+		<header class="mainheader">
+			<h1>Greta Moseley: Software Developer</h1>
+			<div class="imgGreta">
+				<img src="images/greta.jpg">
+			</div>
+			<nav>
+				<ul>
+					<li><a href="#bio">Biography</a></li>
+					<li><a href="#resources">Resources</a></li>
+					<li><a href="#blog">The Mozecode Blog</a></li>
+					<li><a href="#projects">Projects</a></li>
+				</ul>
+			</nav>
+		</header>
+		<div class="container">
+			<section id="bio">
+				<div class="biowrapper">
+					<h2>Professional History</h2>
+					<article>
+						<header>
+							<h3>Forum Services</h3>
+							<h4>Project Manager</h4>
+							<time class="dates">2012-2017</time>
+						</header>
+						<section>
+							
```

Comment:
Be very careful with whitespace. It will inevitably lead to merge conflicts down the road. 

You can change your settings in sublime to automatically trim the whitespace. Just enter this line into your custom settings.
`"trim_automatic_white_space": true,`

Also, a good way to visualize trailing whitespace is to add the Trailing Spaces package to your editor. It highlights any trailing spaces in your code and makes it super easy to stay on top of :)